lambdas/postgres-migration-async-operation/tests/test-index.js

```diff
@@ -0,0 +1,14 @@
+const test = require('ava');
+const sinon = require('sinon');
+const asyncOperations = require('@cumulus/async-operations');
+
+// eslint-disable-next-line unicorn/import-index
+const { handler } = require('../dist/lambda/index');
+
+test('handler calls startAsyncOperation', async (t) => {
+  const stub = sinon.stub(asyncOperations, 'startAsyncOperation').resolves(1);
```

Comment:
Should we use a spy here and ensure it's called with the expected arguments, given the argument translation going on in the lambda is part of the logic being tested?

Reply:
[`e0b53d0` (#2165)](https://github.com/nasa/cumulus/pull/2165/commits/e0b53d0caad6c024a490901ccc2670b669500cb5)

Reply:
👍🏻  s/spy/stub/